src/app/(project)/projects/[project]/drops/Drops.tsx

```diff
@@ -402,35 +420,6 @@ export default function Drops({ project }: DropsPageProps) {
           </>
         )}
       </div>
-      {/* TODO: Fix Modal to show as overlay instead of in footer. */}
-
-      <Modal
-        open={showModal === ShowModal.PAUSE_MINT}
-        setOpen={(open: boolean) => {
-          open ? setShowModal(ShowModal.PAUSE_MINT) : setShowModal(ShowModal.NONE);
-        }}
-      >
-        <Card className="w-[400px]">
-          <Typography.Header size={Size.H2} className="self-start">
-            Pause mint
-          </Typography.Header>
-          <Typography.Header size={Size.H3} className="self-start">
-            Are you sure you want to pause [Name] drop and stop sales?
-          </Typography.Header>
-          <Form className="flex flex-col mt-5">
-            <Button htmlType="submit" className="w-full mt-5">
-              Pause mint
-            </Button>
-            <Button
-              className="w-full mt-5"
-              variant="tertiary"
-              onClick={() => setShowModal(ShowModal.NONE)}
-            >
-              Cancel
-            </Button>
-          </Form>
-        </Card>
-      </Modal>
```

Comment:
We need to show the pause and resume modals on both the drops list and drop detail view so doing as a route isn't ideal especially on the drop detail since children is already used for showing the tab pages (eg holders, mints). For this lets do.


```javascript
<Drop.Pause>
{({ pause }) => (
  <Drop.Resume>
  {({ resume }) => {
   return (
     <>...<>
   )
  }}
  </Drop.Resume>
)}
</Drop.Pause>
```

The drop pause and drop resume components have a modal and track state for the active drop being paused or resumed.

Pass drop id to pause or resume, have it lookup of the details needed for showing the modal (namely the name), providing loading state while fetching, submit the mutation and clear out pause.

You'll need to manually adjust the apollo cache but I can help with that. Hard to write out you can review what I do after.

Reply:
This can be used on drop list or drop detail.